app/celery/scheduled_tasks.py

```diff
@@ -374,3 +379,70 @@ def delete_old_records_from_events_table():
     current_app.logger.info(f"Deleted {deleted_count} historical events from before {delete_events_before}.")
 
     db.session.commit()
+
+
+@notify_celery.task(name="zendesk-new-email-branding-report")
+def zendesk_new_email_branding_report():
+    previous_weekday = datetime.today().date() - timedelta(days=1)
+
+    # If yesterday is a Saturday or Sunday, adjust back to the Friday
+    if previous_weekday.isoweekday() in {6, 7}:
+        previous_weekday -= timedelta(days=(previous_weekday.isoweekday() - 5))
+
+    new_email_brands = (
+        EmailBranding.query.join(Organisation, isouter=True).filter(EmailBranding.created_at >= previous_weekday).all()
+    )
+
+    current_app.logger.info(f"{len(new_email_brands)} new email brands to review since {previous_weekday}.")
+
+    if not new_email_brands:
+        return
+
+    brands_by_organisation = defaultdict(list)
+    brands_with_no_organisation = []
+    for new_brand in new_email_brands:
+        if not new_brand.organisations:
+            brands_with_no_organisation.append(new_brand)
+
+        else:
+            for organisation in new_brand.organisations:
+                brands_by_organisation[organisation].append(new_brand)
+
+    template = jinja2.Template(
+        textwrap.dedent(
+            """
+            <p>There are new email brands to review since {{ yesterday }}.</p><hr>
+            {%- if brands_by_organisation -%}
+            {%- for org, brands in brands_by_organisation.items() -%}
+                <p><a href="https://www.{{ domain }}/organisations/{{ org.id }}/">{{ org.name }}</a>:</p><ul>
+                {%- for brand in brands -%}
+                    <li><a href="https://www.{{ domain }}/email-branding/{{ brand.id }}/edit">{{ brand.name }}</a></li>
+                {%- endfor -%}
+                </ul><hr>
+            {%- endfor -%}
+            {%- endif -%}
+            {%- if brands_with_no_organisation -%}
+                <p>Unassociated brands:</p><ul>
+                {%- for brand in brands_with_no_organisation -%}
+                    <li><a href="https://www.{{ domain }}/email-branding/{{ brand.id }}/edit">{{ brand.name }}</a></li>
+                {%- endfor -%}
+                </ul>
+            {%- endif -%}
+            """
+        ).strip()
+    )
+    message = template.render(
+        domain=current_app.config["NOTIFY_EMAIL_DOMAIN"],
+        yesterday=previous_weekday.strftime("%A %-d %B %Y"),
+        brands_by_organisation=brands_by_organisation,
+        brands_with_no_organisation=brands_with_no_organisation,
+    )
+
+    ticket = NotifySupportTicket(
+        subject="Review new email brandings",
+        message=message,
+        ticket_type=NotifySupportTicket.TYPE_TASK,
```

Comment:
We only use the task ticket type manually when we want to create a reminder for someone on support since they give you a drop down date box with an alert. That probably wouldn't be useful here, so we could stick to the ticket type of `NotifySupportTicket.TYPE_INCIDENT`, which is what we mostly use

Reply:
That's a fair suggestion, will update 👍